rosdep/python.yaml

```
@@ -9505,6 +9505,19 @@ python3-typeguard-pip:
   ubuntu:
     pip:
       packages: [typeguard]
+python3-types-pyyaml:
+  debian: [python3-typeshed]
+  fedora:
+    pip:
+      packages: [types-pyyaml]
```

https://packages.fedoraproject.org/pkgs/python-types-pyyaml/python3-types-pyyaml/

```suggestion
  fedora: [python3-types-pyyaml]
```